ynh_install_ruby/ynh_install_ruby__2

```diff
@@ -117,98 +118,108 @@ ynh_install_ruby () {
     PATH=$(echo $CLEAR_PATH | sed 's@/usr/local/bin:@@')
 
     # Move an existing Ruby binary, to avoid to block rbenv
+    ynh_print_info "Temporarily move the ruby binary"
     test -x /usr/bin/ruby && mv /usr/bin/ruby /usr/bin/ruby_rbenv
+    (
+        set -eE
```

Comment:
I am not very sure about these options. What do you think would be the most relevant?